src/templates/static/legal/tac_sections/affiliate.jsx

```diff
@@ -1,158 +1,358 @@
-import React from 'react';
-import ListNested from '../../../_common/components/list_nested.jsx';
+import React       from 'react';
+import ListNested  from '../../../_common/components/list_nested.jsx';
 
 const Affiliate = () => (
     <div>
-        <h2 data-anchor='affiliate-terms-and-conditions'>{it.L('[_1]\'s Affiliate Programme terms and conditions', it.website_name)}</h2>
+        <h2 data-anchor='affiliate-terms-and-conditions'>{it.L(
+            '[_1]\'s Affiliate Programme terms and conditions',
```

Comment:
do we still need to repeat `terms and conditions` here now that it's moved to be part of t&c?